va/va.go

```diff
@@ -110,6 +124,13 @@ func New(
 		va.log.Printf("Disabling random VA sleeps")
 	}
 
+	sleepTime := os.Getenv(sleepTimeEnvVar)
+	sleepTimeInt, err := strconv.Atoi(sleepTime)
+	if err == nil && sleepTimeInt >= 1 {
+		va.sleepTime = sleepTimeInt
+		va.log.Printf("Setting maximal random VA sleep time to %d seconds", va.sleepTime)
```

Comment:
small nit: I think "maximum" reads a little bit more naturally than "maximal"

Reply:
I get these confused regularly... Thanks for pointing this out!